website/docs/docs/cloud/migration.md

```diff
@@ -11,15 +11,15 @@ dbt Labs is in the process of rolling out a new cell-based architecture for dbt
 
 We're scheduling migrations by account. When we're ready to migrate your account, you will receive a banner or email communication with your migration date. If you have not received this communication, then you don't need to take action at this time. dbt Labs will share information about your migration with you, with appropriate advance notice, when applicable to your account.
 
-Your account will be automatically migrated on its scheduled date. However, if you use certain features, you must take action before that date to avoid service disruptions.
+Your account will be automatically migrated on or after its scheduled date. However, if you use certain features, you must take action before that date to avoid service disruptions.
```

Comment:
If it's "on or after," can we provide a time frame (within a hour/day/week of the date)?

Reply:
@matthewshaver Right now, no. Accounts are eligible for migration when they're scheduled; however, in the window between scheduling and executing the migration, an account can become ineligible for a number of reasons. How long it takes for them to become eligible again depends on what/how much work is required to support the specific use-case.